airbyte-config/init/src/main/resources/seed/source_definitions.yaml

```diff
@@ -8,7 +8,7 @@
 - name: AWS CloudTrail
   sourceDefinitionId: 6ff047c0-f5d5-4ce5-8c81-204a830fa7e1
   dockerRepository: airbyte/source-aws-cloudtrail
-  dockerImageTag: 0.1.2
+  dockerImageTag: 0.1.3
```

Comment:
Please fix out the versions in this file. Here the versions of the connectors have been changed, both for the current ones and for those that you changed in the previous [PR](https://github.com/airbytehq/airbyte/pull/8434). It is necessary to roll back the version changes for connectors that are not related to the current [PR](https://github.com/airbytehq/airbyte/pull/8434), and correctly specify the versions for the connectors with which you work in this [PR](https://github.com/airbytehq/airbyte/pull/8434).